jbmc/src/java_bytecode/java_utils.cpp

```diff
@@ -420,11 +420,13 @@ resolve_inherited_componentt::inherited_componentt get_inherited_component(
 /// \return true if this static field is known never to be null
 bool is_non_null_library_global(const irep_idt &symbolid)
 {
-  static const std::unordered_set<irep_idt> non_null_globals = {
+  static const irep_idt non_null_globals[] = {
```

Comment:
Might as well just do
```
static const irep_idt system_out = ...
static const irep_idt system_err = ...
...
```
in that case

Reply:
Could do - it doesn't seem to make much of a difference, I think? Let me know if you think it would be better to spell them out individually.